codespell_lib/data/dictionary.txt

```diff
@@ -5647,13 +5647,22 @@ deporarily->temporarily
 depracated->deprecated
 depreacte->deprecate
 depreacted->deprecated
+depreacts->deprecates
 deprecatedf->deprecated
 deprectaed->deprecated
 deprectate->deprecate
 deprectated->deprecated
 deprectates->deprecates
 deprectating->deprecating
+deprectation->deprecation
+deprectats->deprecates
 deprected->deprecated
+depretiate->depreciate, deprecate,
```

Comment:
To me deprecate seems like quite a stretch as a suggestion for these.

Reply:
It seems like a stretch but it isn't not because of spelling but more because of misuse. Many people mix up depreciation with deprecation and vice-versa